migrations/20160822191515-processedAt-in-Donation-migration.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.addColumn(
+      'Donations',
+      'processedAt',
```

Comment:
what is the use of `processedAt`? There is already `isProcessed` to know if donation is processed. And there's already an `updatedAt` field. Also the `Expense` model uses a `status` (`PENDING`, `APPROVED`, etc). Should it be similar for donations (also for symmetry) instead of a boolean field?